packages/web-pkg/src/components/SideBar/FileSideBar.vue

```diff
@@ -334,6 +345,39 @@ export default defineComponent({
       sharesStore.setLoading(false)
     }).restartable()
 
+    watch(
+      () => [...unref(panelContext).items, props.isOpen],
+      async () => {
+        if (unref(panelContext).items?.length !== 1) {
+          // don't load additional metadata for empty or multi-select contexts
+          return
+        }
+
+        const resource = unref(panelContext).items[0]
+
+        if (loadVersionsTask.isRunning) {
+          loadVersionsTask.cancelAll()
+        }
+
+        if (
+          !resource.isFolder &&
+          !isProjectSpaceResource(resource) &&
+          !isIncomingShareResource(resource) &&
+          !unref(isPublicFilesLocation) &&
+          !unref(isTrashLocation) &&
+          !unref(isSharedWithOthersLocation) &&
+          !unref(isSharedViaLinkLocation)
```

Comment:
I think you can remove those 2, versions should be available here since we don't have resharing anymore.